vies.go

```diff
@@ -0,0 +1,68 @@
+package gobltin
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+
+	"github.com/go-resty/resty/v2"
+	"github.com/invopop/gobl/cbc"
+	"github.com/invopop/gobl/l10n"
+	"github.com/invopop/gobl/tax"
+)
+
+const viesAPIURL = "https://ec.europa.eu/taxation_customs/vies/rest-api//check-vat-number"
+
+// VIESLookup is a struct that implements the VIES lookup and inherits from TinLookup
+type VIESLookup struct{}
+
+// CheckVatRequest is the request body for the VIES API
+type CheckVatRequest struct {
+	CountryCode l10n.CountryCode `json:"countryCode"`
+	VatNumber   cbc.Code         `json:"vatNumber"`
+}
+
+// CommonResponse is the response body for the VIES API
+type CommonResponse struct {
```

Comment:
This is a generic name for a specific namespace in the public package. If this is in an independent package it'll scale a bit better.